string action
string parameters
---
string action_uuid
string error_msg
